docs/sources/clients/promtail/scraping.md

```diff
@@ -345,6 +345,31 @@ For sending messages via UDP:
 *.* action(type="omfwd" protocol="udp" target="<promtail_host>" port="<promtail_port>" Template="RSYSLOG_SyslogProtocol23Format")
 ```
 
+## Azure Event Hub
+
+Promtail supports reading messages from Azure Event Hubs.
+Targets can be configured using the `azure_event_hub` stanza:
+
+```yaml
+- job_name: azure_event_hub
+  azure_event_hub:
+    group_id: "mygroup"
+    fully_qualified_namespace: my-namespace.servicebus.windows.net:9093
+    connection_string: "my-connection-string"
+    event_hubs:
+      - event-hub-name
+    labels:
+      job: azure_event_hub
+  relabel_configs:
+    - action: replace
+      source_labels:
+        - __azure_event_hub_category
+      target_label: category
+```
+
+Only the `fully_qualified_namespace`, `connection_string` and `event_hubs` are required.
+Read the [configuration]({{< relref "configuration/#azure-event-hub" >}}) section for more information.
```

Comment:
```suggestion
Read the [configuration]({{< relref "../../configuration/#azure-event-hub" >}}) section for more information.
```

Reply:
Hi @JStickler,

I have a question about this suggestion 😃 

This PR adds a new target to Promtail, so I decided to keep documentation within Promtail docs. Similar to the other targets in this file.

If I understand correctly, "../../configuration" is a Loki configuration, not the Promtail one.

Could you please elaborate a little bit more on your suggestion?

Reply:
Ugh, I'm sorry, that's my mistake.  Ignore this suggestion.

Reply:
@JStickler, because it was already merged, I created a PR to revert it: https://github.com/grafana/loki/pull/8904. Please take a look when you have time.